controller/controller.cpp

```diff
@@ -1038,12 +1038,72 @@ void Controller::loadProject(const QString &projectName, const QPointer<QFileDev
 
         //load bundle templates
         QList<QJsonObject> templates = this->loadBundleTemplates();
-        if(!templates.isEmpty()
-                && !this->job->getActiveBundleSystem().isNull()) {
+        if(!templates.isEmpty()) {
             QJsonObject bundleTemplate = templates.first(); // only one should exists
-            //load template
-            //this->loadBundleTemplate(bundleTemplate);
-            this->job->getActiveBundleSystem()->setBundleTemplate(bundleTemplate);
+
+            if(this->job->getActiveBundleAdjustment().isNull()) { // no bundle result was saved
+                // load plugin and set plugin to bundle system "Bundle01"
+                // see MainWindow::initBundleView()
+                QList<QPointer<FeatureWrapper> > bundleSystems = this->job->getFeaturesByName("Bundle01");
+                if(!bundleSystems.isEmpty()) {
+                    QPointer<CoordinateSystem> bundleSystem = bundleSystems.first()->getCoordinateSystem();
+
+                    QString bundleName, pluginName;
+                    bundleName = bundleTemplate.value("plugin").toObject().value("name").toString();
+                    pluginName = bundleTemplate.value("plugin").toObject().value("pluginName").toString();
+
+                    //get and check plugin information
+                    sdb::Plugin plugin = SystemDbManager::getPlugin(pluginName);
+                    if(plugin.id == -1){
+                        this->log(QString("No plugin available with the name %1").arg(pluginName), eErrorMessage, eMessageBoxMessage);
+                        return;
+                    }
+
+                    //load bundle plugin
+                    QPointer<BundleAdjustment> bundlePlugin = PluginLoader::loadBundleAdjustmentPlugin(plugin.file_path, bundleName);
+                    if(bundlePlugin.isNull()){
+                        this->log(QString("Cannot load bundle template %1").arg(bundleTemplate.value("name").toString()), eErrorMessage, eMessageBoxMessage);
+                        return;
+                    }
+
+                    //get bundle system
+                    if(bundleSystem.isNull()){
```

Comment:
Ist in 1046/47 für mich sichergestellt, falls nicht sollte dieser Block auch vorher abgefragt werden